modules/message.js

```diff
@@ -0,0 +1,19 @@
+function showMsg(msg) {
+  msg = document.querySelector('.success-message');
+  msg.style.visibility = 'visible';
+  msg.textContent = 'Your book added successfully!';
+  setTimeout(() => {
+    msg.textContent = '';
+  }, 3000);
+}
```

Comment:
- [ ] Please change this and **all other** regular functions to es6 arrow function syntax.